tests/fiveshell/shell_example.py

```diff
@@ -3,13 +3,17 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 import pandas as pd
+import os
 import sys
-sys.path.append("../")
+sys.path.append(os.path.abspath("../../"))
```

Comment:
none of this would be necessary if the package was setup correctly.

Reply:
I am tempted to reject this commit because you definitely don't want to do this. But since you were already doing a form of this, we're going to leave it in place for now. But we definitely need to revisit in a couple weeks--after implementing click and getting your package set up properly--and make sure we delete all the appending to the sys.path!